lxd/cluster/config.go

```diff
@@ -246,6 +247,7 @@ var ConfigSchema = config.Schema{
 	"images.auto_update_cached":      {Type: config.Bool, Default: "true"},
 	"images.auto_update_interval":    {Type: config.Int64, Default: "6"},
 	"images.compression_algorithm":   {Default: "gzip", Validator: validate.IsCompressionAlgorithm},
+	"images.default_architecture":    {Validator: defaultArchValidator},
```

Comment:
With the step mentioned in the previous commit, you could then use `validate.IsArchitecture` here.

Reply:
@stgraber same problem here as in my previous answer.